### ISSUE TITLE
Tags not in system for proj 38

### ISSUE THREAD

**Issue by zcrysler** [jbrzusto/TO_DO#110]
Most of the tags are now in the system, but some are still missing from James Bay:

Proj 38 tags 90-166 BI 4.7 Job status 495835 says they are already registered, but they aren't in the project on motus.org - there are some registered as NTQB-3-2 with a different burst rate with those IDs

**Comment by jbrzusto** [comment]
This is the motus.org server (i.e. not mine) refusing to register these tags, apparently correctly:
these tags have already been registered to that project, with August, 2017 deployment dates.
If this is a new set of identical tags, the tagreg.txt file for that upload needs to be given a new
deployment date, so that these tags end up requesting a different dateBin.
The file currently reads:
```
motusProjID: 38
tagModel: ANTC-M4-2S
nomFreq: 166.38
species:
deployDate: 2017-08-01
                    ^^^^^^^^^^^
```
Moreover, the recordings in this uploaded archive all have dates from July 2017.

Are you trying to move these already registered tags to a different project?
Or did someone just upload the wrong file, or just send the same file without
bothering to actually record their tags (a bad idea, since then there's no guarantee
the new batch of tags is actually behaving as advertised).

**Comment by zcrysler** [comment]
So the user tried uploading these to project 38 back in July but many of them failed.  I now need to move some of them to project 47, but since they don't exist in the Motus database I can't move them, and because this issue was already logged I didn't want to try registering the missing tags directly to proejct 47 because I was worried it would result in duplicate tags in the system (even though I don't see them in there at all right now).  So this morning I uploaded the missing tags (I have the original .wav files that were uploaded in July), along with the original tagreg file and got the error message I included above.

So what's the best approach?  Should I make a new tagreg file and upload the recordings directly to project 47, or is it possible to figure out why they are being refused on the motus server side?

**Comment by jbrzusto** [comment]
Here's what motus.org is returning as registered tags for project 38 in 2017, when called
via the API.  Are you seeing these on the motus.org website?  If not, that's a bug on their side.
Using this query from the meta cache (which is refreshed daily from the motus.org API):
```sql
select mfgID, round(period,1) as period, tagID as motusTagID, dateBin from tags 
where projectID=38 and dateBin >= "2017" and mfgID between "109" and "166" order by mfgID;
```
I get this list, which includes all the recordings uploaded in that archive:

mfgID|period|motusTagID|dateBin
---|---|---|---
109|9.7|25375|2017-3
110|9.7|25376|2017-3
111|9.7|25377|2017-3
112|9.7|25378|2017-3
113|9.7|25379|2017-3
114|9.7|25380|2017-3
115|9.7|25381|2017-3
116|9.7|25382|2017-3
117|9.7|25383|2017-3
118|9.7|25384|2017-3
119|9.7|25385|2017-3
120|9.7|25386|2017-3
121|9.7|25764|2017-3
122|9.7|25387|2017-3
123|9.7|25388|2017-3
124|9.7|25389|2017-3
125|9.7|25390|2017-3
126|9.7|25391|2017-3
127|9.7|25392|2017-3
128|9.7|25393|2017-3
129|9.7|25394|2017-3
130|9.7|25395|2017-3
131|9.7|25396|2017-3
132|9.7|25397|2017-3
133|9.7|25398|2017-3
134|9.7|25399|2017-3
135|9.7|25400|2017-3
136|9.7|25401|2017-3
137|9.7|25765|2017-3
138|9.7|25402|2017-3
139|9.7|25403|2017-3
140|9.7|25404|2017-3
141|9.7|25405|2017-3
142|9.7|25406|2017-3
143|9.7|25407|2017-3
144|9.7|25408|2017-3
145|9.7|25409|2017-3
146|9.7|25410|2017-3
147|9.7|25411|2017-3
148|9.7|25412|2017-3
149|9.7|25766|2017-3
150|9.7|25413|2017-3
151|9.7|25414|2017-3
152|9.7|25415|2017-3
153|9.7|25416|2017-3
154|9.7|25417|2017-3
155|9.7|25418|2017-3
156|9.7|25419|2017-3
157|9.7|25767|2017-3
158|9.7|25420|2017-3
159|9.7|25421|2017-3
160|9.7|25768|2017-3
161|9.7|25422|2017-3
162|9.7|25423|2017-3
163|9.7|25424|2017-3
164|9.7|25425|2017-3
165|9.7|25426|2017-3
166|9.7|25427|2017-3

**Comment by zcrysler** [comment]
Yes I can see those, but those are for NTQB-3-2 with BI 9.7.  The recordings I uploaded are for ANTC-M4-2S and have BI 4.7

So based on the tagreg shouldn't they be allowed to register since it's a different tag model?

Also, I've checked a number of the .wav files I uploaded, and they all have a BI of 4.7 so they shouldn't match the ones with a BI of 9.7

**Comment by jbrzusto** [comment]
Sorry, missed that.  If there are two tags for the same ID and dateBin but with different BIs, then
one of them needs to have ".1" added to the tag ID in the name of the .wav file, as described in the
tag registration instructions [here](https://sensorgnome.org/VHF_Tag_Registration/Use_of_tags_with_duplicate_IDs_within_a_project)
This is meant to strongly encourage users to physically mark such pairs
of tags so that there won't be any confusion about which one was deployed on what organism.
I'll rename the files you uploaded and re-run that job.

**Comment by zcrysler** [comment]
Ah sorry I should have known that!  Thanks, the tags are all there now
